README.md

```diff
@@ -72,9 +97,15 @@ To build Cuberite and the C++ Minecraft client:
 make
 ```
 
-### Run the Cuberite instance
+# Interacting with the Agent
+
+If you did a local install, run the following commands from the cloned repository directory.
+
+If you installed CraftAssist using Docker, run these commands in terminals inside of your Docker container (Docker has been configured to open shells in the correct directory). Also, Docker users should use the `python3` command instead of `python`.
```

Comment:
Actually you can symlink `/usr/bin/python` to `python3` so that `python3` is not needed. This ensures consistency between docker option and local option.

Just add something like `RUN rm -f /usr/bin/python && ln -s /usr/bin/python3 /usr/bin/python` to `Dockerfile.client`. Note that doing this may have side effect so please add this line near the end of docker file (maybe right above '`ENV  LANG C.UTF-8`'). However, since you are using Ubuntu 18.04 it should be just fine.

Reply:
I aliased `python` to `python3` instead  - that should allow for consistency while not removing access to python 2.7 to anything that might need it.